frontend/assets/stylesheets/components/_packages.scss

```diff
@@ -0,0 +1,365 @@
+/* ==========================================================================
+   Packages
+   ========================================================================== */
+
+.package-container {
+    border-top: rem(1px) solid transparent;
```

Comment:
Might leave that in and do a find-and-replace in a separate PR. We appear to have a lot of `border-top: rem(1px)`